### ISSUE TITLE
Tests fail with pandas==1.1(metadata lost on groupby)

### ISSUE THREAD

**Issue by Flix6x** [SeitaBV/timely-beliefs#26]
Until [this issue](https://github.com/pandas-dev/pandas/issues/29442) is fixed, timely-beliefs only supports pandas<1.1.

**Comment by nhoening** [comment]
The pandas fix has 1.1.1 as target, so 1.1.0 wouldn't work. Or they backport the fix.

Am 15. September 2020 20:05:25 MESZ schrieb Felix Claessen <notifications@github.com>:
>Until [this issue](https://github.com/pandas-dev/pandas/issues/29442)
>is fixed, timely-beliefs only supports pandas<1.1.
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/SeitaBV/timely-beliefs/issues/26

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by Flix6x** [comment]
The fix was released as part of pandas 1.1.5. There was a [backport](https://github.com/pandas-dev/pandas/pull/37628), too, but our relevant tests still fail. @nhoening can you comment if my procedure below is correct? I am probably misunderstanding how backports actually work. Basically, I thought a backport of a fix would lead to that fix being present in the code of relevant older versions as available on pypi, perhaps as a developer release. However, the latest release of 1.1.4 on pypi is on October 30th, while the backport was merged on November 4th.

Here I try to install a previous version of pandas with the backport supposedly in place:

```
pip install pandas==1.1.4 --no-cache --force-reinstall
```

Then I run the test that I would expect to pass (if this focused test fails, other integration tests fail, too, of course):

```
pytest -k test_groupby_retains_metadata
```
This test, and all others, too, are passing against pandas==1.1.5 on #23 .

**Comment by nhoening** [comment]
It was indeed not obvious to find, but they had it set on 1.2 in the beginning of the PR, but then backported it to 1.1.5 when they were done.
You can see this [here](https://github.com/pandas-dev/pandas/pull/37461#pullrequestreview-522002408) (click "show resolved"), where they removed the release note entry from the 1.2 part (comment: "Force pushed to drop this commit").
